evaluations/qa/medquad_evaluator.py

```diff
@@ -0,0 +1,72 @@
+from typing import List
+
+import evaluate
+import numpy as np
+
+from prompt_templates.prompt_abstract import Prompt
+from evaluations.causal_llm_evaluators import CausalLanguageModelEvaluator
+from prompt_templates.qa.qa_prompt import MedQuADQuestionAnswerPromptV1, MedQuADQuestionAnswerPromptBaseV1
+
+
```

Comment:
I am not sure if you need this custom metric since you are using the standard rogue, bleu and meteor